tasks/MLSMapLoader.hpp

```diff
@@ -0,0 +1,34 @@
+#ifndef ENVIRE_MAPS_MLSMAPLOADER_TASK_HPP
+#define ENVIRE_MAPS_MLSMAPLOADER_TASK_HPP
+
+#include "envire_maps/MLSMapLoaderBase.hpp"
+
+#include <pcl/common/common.h>
+#include <maps/grid/Index.hpp>
+
+namespace envire_maps
+{
+    class MLSMapLoader : public MLSMapLoaderBase
+    {
+    friend class MLSMapLoaderBase;
+    protected:
+        bool mMapLoaded;
+
+    protected:
+        virtual void publishMap();
+
+        virtual void createMLS(const pcl::PointCloud<pcl::PointXYZ>& pc,
```

Comment:
Defining `createMLS` and `writeMLS` as pure virtual didn't work?

Reply:
I actually did not try that. Can you define pure virtual orogen-tasks? I figured that each orogen-task is build into a default-deployment and thus needs to be instantiated.

Reply:
I just verified this:

`/home/skasperski/Robotics/rock/robdekon/install/include/rtt/Component.hpp:100:20: error: invalid new-expression of abstract class type ‘envire_maps::MLSMapLoader’`

Reply:
Ok good to know :+1: